team.cpp

```diff
@@ -1,14 +1,25 @@
 #include "team.h"
 
 #include <QJsonObject>
+#include <QJsonArray>
+
+#include "group.h"
 
 namespace DJ {
 namespace Model {
-Team::Team(QJsonObject team, QObject *parent) : QObject(parent) {
+Team::Team(QJsonObject team, QHash<QString, Group *> groups, QObject *parent) : QObject(parent) {
     this->id = team.value("id").toString();
     this->name = team.value("name").toString("UNKNOWN");
     this->affilation = team.value("affilation").toString("UNKNOWN");
     this->nationality = team.value("nationality").toString("UNKNOWN");
+    QJsonArray groups_ids = team.value("group_ids").toArray();
+    QString groupId = groups_ids[0].toString();
```

Comment:
I think this would crash if a team belongs to no group? Although in DOMjudge this is currently not possible, I think, we can easily add a `.size() > 0` here, right?

Reply:
Okay I will take care of that!